assets/javascript/game.js

```diff
@@ -0,0 +1,590 @@
+        $('#emeny-attack-back').append(parseInt(opp.AttackBack));
+        console.log(parseInt(opp.AttackBack));
+        $('#player-health').empty();
+        $('#player-health').append(parseInt(character.health = parseInt(character.health) - parseInt(opponent.AttackBack)));
+        console.log(parseInt(character.health = parseInt(character.health) - parseInt(opponent.AttackBack)))
+        attackSound.play();
+    }
+}
+
+// "Do. Or do not. There is no try." Function
+function chooseOpp2(opp2)   {
+    if (chosenEnemy2 == false)  {
+        opponent2 = opp2;
+        chosenEnemy2 = true; 
+        $('#results').text("YOU'VE BEEN ATTACKED! STRIKE BACK PADAWAN!")
+        $('#defenders-line').hide()
+        $('#emeny-name').empty();
+        $('#emeny-health').empty();
+        $('#emeny-attack').empty();
+        $('#emeny-attack-back').empty();
+        $('#defenders-line2').append(opp2.defendImage)
+        $('#emeny-name').append(opp2.name);
+        $('#emeny-health').append(parseInt(opp2.health));
+        console.log(parseInt(opp2.health));
+        $('#emeny-attack').append(parseInt(opp2.attack));
+        console.log(parseInt(opp2.attack));
+        $('#emeny-attack-back').append(parseInt(opp2.AttackBack));
+        console.log(parseInt(opp2.AttackBack));
+        $('#player-health').empty();
+        $('#player-health').append(parseInt(character.health = parseInt(character.health) - parseInt(opponent2.AttackBack)));
+        console.log(parseInt(character.health = parseInt(character.health) - parseInt(opponent2.AttackBack)))
+        attackSound.play();
+    }
+}
+
+//"Final Battle With The CYST Lord" Function
+function chooseOpp3(opp3)   {
+    if (chosenEnemy3 == false)  {
+        opponent3 = opp3;
+        chosenEnemy3 = true; 
+        $('#results').text("YOU'VE BEEN ATTACKED! STRIKE BACK PADAWAN!")
+        $('#defenders-line2').hide()
+        $('#emeny-name').empty();
+        $('#emeny-health').empty();
+        $('#emeny-attack').empty();
+        $('#emeny-attack-back').empty();
+        $('#defenders-line3').append(opp3.defendImage)
+        $('#emeny-name').append(opp3.name);
+        $('#emeny-health').append(parseInt(opp3.health));
+        console.log(parseInt(opp3.health));
+        $('#emeny-attack').append(parseInt(opp3.attack));
+        console.log(parseInt(opp3.attack));
+        $('#emeny-attack-back').append(parseInt(opp3.AttackBack));
+        console.log(parseInt(opp3.AttackBack));
+        $('#player-health').empty();
+        $('#player-health').append(parseInt(character.health = parseInt(character.health) - parseInt(opponent3.AttackBack)));
+        console.log(parseInt(character.health = parseInt(character.health) - parseInt(opponent3.AttackBack)))
+        attackSound.play();
+    }
+}
+
+//"Take Ass Kick Names" Button (I know it's an Avengers: Infinity War reference but who cares since we're like Peter Pan role playing here)
+$('#attack-button').on("click", function(){
```

Comment:
All three attack listeners are really similar, there's probably a way to simplify writing them.